EventFilter/CSCRawToDigi/plugins/CSCViewDigi.cc

```diff
@@ -1,14 +1,60 @@
-/**
+/**\class CSCViewDigi CSCViewDigi.h
```

Comment:
These comments do not hold any more, now that you have merged the two files

Reply:
Fixed.